### DISCUSSION TITLE
s/fl3oix7xg9gvrj9g--a5-g

### DISCUSSION THREAD

**Discussion by giscus** [ops-coffee/blog#7] [Announcements]
# s/fl3oix7xg9gvrj9g--a5-g

记一次诡异的故障排查经历

https://blog.ops-coffee.cn/s/fl3oix7xg9gvrj9g--a5-g

**Comment by Gailun90** [comment]
我觉得还能进一步追查 ”tomcat8的umask默认改成了0027“ 是因为什么
